searx/__init__.py

```diff
@@ -20,7 +20,6 @@
 from os import environ
 from os.path import realpath, dirname, join, abspath, isfile
 
-
```

Comment:
I think this file does not need to be touched.

Reply:
Fixed baff1cbbab8a72155823c3186971a5f56d7a92f2